<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Box1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Box7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: white;
	color: black;
	font-size: 60px;
	font-weight: bold;
}
QPushButton: hover
{
background-color: black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="winner">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit
{
	color: black;
	background-color: white;
	border-radius: 3px;
	font-weight: bold;
	font-size: 32px;
	text-align: center;
}
</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
